docs/quickstart.md

```diff
@@ -0,0 +1,139 @@
+# Quickstart
+
+This section guides you through all the necessary steps to create a workload Kubernetes cluster on top of the OpenStack infrastructure. The guide describes a path that utilizes the [clusterctl] CLI tool to manage the lifecycle of a CAPI management cluster and employs [kind] to create a local non-production management cluster.
+
+Note that it is a common practice to create a temporary, local [bootstrap cluster](https://cluster-api.sigs.k8s.io/reference/glossary#bootstrap-cluster) which is then used to provision a target [management cluster](https://cluster-api.sigs.k8s.io/reference/glossary#management-cluster) on the selected infrastructure.
+
+## Prerequisites
+
+- Install [Docker] and [kind]
+- Install [kubectl]
+- Install [clusterctl]
+- Install [go]  # installation of the Go package `envsubst` is required to enable the expansion of variables specified in CSPO and CSO manifests.
+
+## Initialize the management cluster
+
+Create the kind cluster:
+
+```bash
+kind create cluster
+```
+
+Transform the Kubernetes cluster into a management cluster by using `clusterctl init` and bootstrap it with CAPI and Cluster API Provider OpenStack ([CAPO]) components:
+
+```bash
+# Enable Cluster Class CAPI experimental feature
+export CLUSTER_TOPOLOGY=true
+
+# Install CAPI and CAPO components
+clusterctl init --infrastructure openstack
+```
+
+### Create a secret for OpenStack access
+
+To enable communication between the CSPO and the Cluster API Provider for OpenStack (CAPO) with the OpenStack API, it is necessary to generate a secret containing the access data (clouds.yaml).
+Ensure that this secret is located in the identical namespace as the other Custom Resources.
+
+> [!NOTE]  
+> The default value of `cloudName` is configured as `openstack`. This setting can be overridden by including the `cloudName` key in the secret. Also, be aware that the name of the secret is expected to be `openstack` unless it is not set differently in OpenStackClusterStackReleaseTemplate in `identityRef.name` field.
+
+```bash
+kubectl create secret generic openstack --from-file=clouds.yaml=path/to/clouds.yaml
+
+# Patch the created secrets so they are automatically moved to the target cluster later.
```

Comment:
We can use `kubectl label` instead:
`kubectl label secret openstack clusterctl.cluster.x-k8s.io/move=`

Reply:
fixed: c380e5fa26ad5d4725148dfaf08bed0f842afcc8